src/ResourceManager/Sql/Commands.Sql/Server/Cmdlet/GetAzureSqlServer.cs

```diff
@@ -26,9 +29,19 @@ namespace Microsoft.Azure.Commands.Sql.Server.Cmdlet
     /// Defines the Get-AzureRmSqlServer cmdlet
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlServer", ConfirmImpact = ConfirmImpact.None, SupportsShouldProcess = true)]
-    public class GetAzureSqlServer : AzureSqlServerCmdletBase, IModuleAssemblyInitializer
+    public class GetAzureSqlServer : AzureSqlCmdletBaseBase<IEnumerable<AzureSqlServerModel>, AzureSqlServerAdapter>, IModuleAssemblyInitializer
     {
         /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
```

Comment:
can you <remarks> or some comment explaining why BaseBase when you have ResourceGroup param? I understand that's it's for `Mandatory=false` when looking at this change in isolation, but it won't be obvious when reading this code later